supplierProduct/tests.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+from supplierProduct.models import supplierProduct, Product, Supplier
+
+supID0 = 123
+supID1 = 222
+supID2 = 127
+
+
+class TestSupProduct:
+
+    @pytest.fixture
+    def creat_product(self):
+        prod0 = Product(qr_code="AQMLS5C369VMSL7FOVNX5VLSX0ASSD22", product_name="APPLE", description="sweet!!")
+        prod1 = Product(qr_code="A0ABG9X6G6DMSL7FOVNX5VLSX0ASSQ38", product_name="CARROT", description="yummy!!")
+        prod2 = Product(qr_code="SS6G33X48V8MSL7FOVNX5VLSX0ASSRRR", product_name="BANANA", description="great!!")
+        prod0.save_product()
+        prod1.save_product()
+        prod2.save_product()
+        return [prod0, prod1, prod2]
+
+    @pytest.fixture
+    def creat_supplier(self):
+        sup0 = Supplier(
+            user_name="ali22",
+            first_name="ali",
+            last_name="ab",
+            password="2345678",
+            user_type="supplier")
+        sup1 = Supplier(
+            user_name="max00",
+            first_name="Max",
+            last_name="hani",
+            password="123456789",
+            user_type="supplier")
+        sup2 = Supplier(
+            user_name="rawanab",
+            first_name="rawan",
+            last_name="abu",
+            password="0000000",
+            user_type="supplier")
+        sup0.save()
+        sup1.save()
+        sup2.save()
+        return [sup0, sup1, sup2]
+
+    @pytest.fixture
+    def creat_supProd(self, creat_product, creat_supplier):
+        supprod0 = supplierProduct(
```

Comment:
dont think other typo mistakes are crucial but i suggest changing supprod0 -> supProd0, to make it more readable.
same for the rest of the code in this method.

Reply:
I think it is okay